src/connector/src/parser/util.rs

```diff
@@ -69,9 +67,7 @@ macro_rules! only_parse_payload {
         if let Some(payload) = $payload {
             $self.parse_inner(payload, $writer).await
         } else {
-            Err(RwError::from(ErrorCode::InternalError(
-                "Empty payload with nonempty key".into(),
-            )))
+            risingwave_common::bail!("empty payload with non-empty key")
```

Comment:
```suggestion
            bail!("empty payload with non-empty key")
```

Reply:
Since it'll be expanded, I prefer qualified path here to make it self-contained.

Reply:
Oh, didn't realize it's in a macro.